lang/test/org/partiql/lang/eval/EvaluatingCompilerExecTests.kt

```diff
@@ -0,0 +1,302 @@
+package org.partiql.lang.eval
```

Comment:
I decided to put the tests in the `eval` sub-package rather than `examples` for the better flexibility in testing. I can also add a smaller example of adding a stored procedure similar to the [`CustomFunctionsExample`](https://github.com/partiql/partiql-lang-kotlin/blob/master/examples/src/kotlin/org/partiql/examples/CustomFunctionsExample.kt) if that would be useful.

Reply:
> I can also add a smaller example of adding a stored procedure similar to the CustomFunctionsExample if that would be useful.
 
I do think having one implementation of `StoredProcedureBase` there would be helpful.  Thanks.

Reply:
Added an example to the `examples` sub-package.